### ISSUE TITLE
scale on data augmentation

### ISSUE THREAD

**Issue by guuusss** [ClementPinard/FlowNetPytorch#57]
HI,thank you for your outstanding work! I have some questions on data augmentation.
I find you do not add scale and chromatic augmentation  to data augmentation ,is it sacrifice performance on chairs or sintel?
And,the hyper-parameter used in data augmentation of you project is different from the original paper,for example, in the original flownet paper,it translates from a the range [−20%; 20%] of the image width for x and y; Is it important for the training?

**Comment by ClementPinard** [comment]
Data augmentation routines are admittedly different from original paper. They apply a large number of different operation, and I didn't take time to implement them all. I found that just applying these ones did suffice to get good results so I didn't make the rest, like chromatic augmentation. I actually wrote something on it, but didn't use it : see RandomColorWarp in flow_transforms.

If you want to test it, I'd be happy to accept pull requests ! ;)

**Comment by jeffbaena** [comment]
Hi @guuusss, I am also trying to replicate the original data augmentation. By any chance did you manage to replicate the scripts?

It would be very useful for me,

Thanks,
Stefano

**Comment by guuusss** [comment]
Hi @jeffbaena, I am sorry that i fail to replicate the [original caffe data augmenation](https://github.com/NVlabs/PWC-Net/blob/864d2e9fb3f60f9af99f4ea72943b1ff5d472202/Caffe/model/PyCaffe/pwc_net_utils.py) for pytorch.
In my project, I choose the augmentation part of this [project](https://github.com/ucbdrive/hd3) instead, as it works fine on MPI-Sintel test set, whose epe ranks 9 on MPI-Sintel Final set. or you can just adjust original caffe data augmentation into your pytorch project :)

**Comment by jeffbaena** [comment]
Thanks so much for your prompt response!
